### PR TITLE
chore: add rabobank to adopters and vertical page

### PR DESCRIPTION
**Pull request by hacktivist123**
This PR adds Rabobank to the financial services vertical page and the adopters page

### CODE REVIEW

File: src/pages/industries/financial-services.jsx [hunk 1]
```diff
@@ -188,9 +189,27 @@ const financialResources = [
...
       ' Form3 integrated Cilium into their platform as their primary solution for networking, security, and observability, enabling them to meet their FPS gateway business requirement of seamlessly switching between data centers without any downtime.',
   },
+  {
+    imageSrc: RabobankOfficeImage,
+    imageAlt: 'Rabobank office building',
+    title: 'Self-service, Zero Trust Network Security',
+    buttonLink: 'https://www.cncf.io/case-studies/rabobank/',
+    buttonText: 'Read The Case Study',
+    description:
+      'Rabobank turned to Cilium as their preferred network security and observability solution, enabling them to implement zero-trust networking and create a self-service environment for their teams.',
```
Comment (xmulligan) [inline src/pages/industries/financial-services.jsx:199]: Can you make this more specific to the use case and business outcome?

**Comment by xmulligan** [comment]
Can you also add to https://cilium.io/use-cases/network-policy/?

**Comment by hacktivist123** [comment]
Alright, will make the updates and revert.

File: src/pages/industries/financial-services.jsx (hunk 1 above)
Comment (hacktivist123) [reply src/pages/industries/financial-services.jsx:199]: Okay, didn't want it to get too long. Will update it

**Comment by hacktivist123** [comment]
Updated @xmulligan

File: src/pages/industries/financial-services.jsx [hunk 2]
```diff
@@ -188,9 +189,27 @@ const financialResources = [
...
       ' Form3 integrated Cilium into their platform as their primary solution for networking, security, and observability, enabling them to meet their FPS gateway business requirement of seamlessly switching between data centers without any downtime.',
   },
+  {
+    imageSrc: RabobankOfficeImage,
+    imageAlt: 'Rabobank office building',
+    title: 'Self-service, Zero Trust Network Security',
+    buttonLink: 'https://www.cncf.io/case-studies/rabobank/',
+    buttonText: 'Read The Case Study',
+    description:
+      'Rabobank leverage Cilium to enable zero-trust networking, significantly improving the security and enabling self-service in their financial API platform',
```
Comment (xmulligan) [inline src/pages/industries/financial-services.jsx:199]:
```suggestion
      'Rabobank leverages Cilium to enable zero-trust networking, significantly improving the security and enabling self-service in their financial API platform',
```

**Comment by hacktivist123** [comment]
updated

**Pull request merged**